### PR TITLE
Update some links from RFC 7234 to RFC 9111

### PR DESCRIPTION
**Pull request by LPardue**
### Description

There were some specific links to RFC 7234, these can be updated.

### Motivation

There are new revisions of HTTP specs.

### CODE REVIEW

File: files/en-us/web/http/headers/cache-control/index.md [hunk 1]
```diff
@@ -453,6 +453,6 @@ Alternatively, `Clear-Site-Data` can clear a browser cache for a site. But be ca
 - [Caching Tutorial for Web Authors and Webmasters](https://www.mnot.net/cache_docs/)
 - [Caching best practices & max-age gotchas](https://jakearchibald.com/2016/caching-best-practices/)
 - [Cache-Control for Civilians](https://csswizardry.com/2019/03/cache-control-for-civilians/)
-- [RFC 7234 – Hypertext Transfer Protocol (HTTP/1.1): Caching](https://httpwg.org/specs/rfc7234.html)
+- [RFC 9110 – HTTP Caching](https://httpwg.org/specs/rfc9111.html#calculating.freshness.lifetime)
```
Comment (teoli2003) [inline files/en-us/web/http/headers/cache-control/index.md:456]: There is an incoherence here between the label _RFC 9110 – HTTP Caching_ and the link (a deep link to RFC 9111)

I think this is the correct link:
```suggestion
- [RFC 9111 – HTTP Caching](https://httpwg.org/specs/rfc9111.html)
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by Josh-Cena** [comment]
Hi @LPardue could you confirm with https://github.com/mdn/content/pull/20747#discussion_r971555542?

File: files/en-us/web/http/headers/cache-control/index.md (hunk 1 above)
Comment (LPardue) [reply files/en-us/web/http/headers/cache-control/index.md:456]: whoops, yes good catch, your suggestion is correct and we should go with that

**Comment by LPardue** [comment]
thanks for the ping. The suggestion is correct, please commit it at your pleasure.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**